### ISSUE TITLE
Better error message when snapshot creation is blocked by merging origin

### ISSUE THREAD

**Issue by bmr-cymru** [snapshotmanager/snapm#40]
Snapshot manager cannot create a new snapshot when a merge is currently scheduled for one or more of the origin volumes making up the set. Right now this fails with the generic "Could not find snapshot provider for XYZ":

```bash
root@localhost:~# snapm snapset create testset / /home /var
ERROR - Command failed: Could not find snapshot provider for /
```

This is because the `lv_attr` flags for the LV indicate a merge is in progress:

```bash
root@localhost:~# lvs
  LV   VG     Attr       LSize   Pool Origin Data%  Meta%  Move Log Cpy%Sync Convert
  home fedora Owi-aos---   1.00g                                                    
  opt  fedora Owi-aos--- 512.00m                                                    
  root fedora Owi-aos---   6.00g                                                    
  srv  fedora Owi-aos--- 512.00m                                                    
  var  fedora Owi-aos---   2.00g 
```

(note 'O' as first character of attrs).

Detect this case and respond with a better error message.

**Comment by bmr-cymru** [comment]
Fixed in commit 88d778bb6414720c51502d721fbafa936cd877a0.

**Comment by bmr-cymru** [comment]
```
root@localhost:~/src/git/snapm# snapm snapset create -br backup / /home /opt /srv /var
SnapsetName:      backup
MountPoints:      /, /home, /opt, /srv, /var
NrSnapshots:      5
Time:             2024-06-17 21:03:44
UUID:             e80b3a90-cc8c-56bb-a33a-6f54953d209f
Status:           Active
Autoactivate:     yes
Bootable:         yes
BootEntries:
  SnapshotEntry:  01f94f6
  RevertEntry:    884fe1c

root@localhost:~/src/git/snapm# snapm snapset revert backup
WARNING - Snaphot set backup origin is in use: reboot required to complete revert
Boot into 'Revert backup 2024-06-17 21:03:44 (6.8.9-300.fc40.x86_64)' to continue

root@localhost:~/src/git/snapm# snapm snapset create -br backup / /home /opt /srv /var
ERROR - Command failed: Snapshot revert is in progress for fedora/root
```
